src/plugins/elasticsearch/index.js

```diff
@@ -19,7 +19,7 @@ module.exports = function ({ Plugin }) {
         username: string(),
         password: string(),
         shardTimeout: number().default(0),
-        requestTimeout: number().default(30000),
+        requestTimeout: number().default(300000),
```

Comment:
I'm not sure we want the default value to be 5 mins. The 30s default seems to be fine for most use cases, and now that the configure value is actually being used, if users need a longer timeout, they can add the `elasticsearch.requestTimeout` value to their config.